tests/fast-integration/utils/index.js

```diff
@@ -491,6 +491,18 @@ function waitForSubscription(name, namespace = "default", timeout = 180000) {
   );
 }
 
+function waitForClusterServiceBroker(name, timeout = 90000) {
+  return waitForK8sObject(
+      `/apis/servicecatalog.k8s.io/v1beta1/clusterservicebrokers`,
+      {},
+      (_type, _apiObj, watchObj) => {
+        return watchObj.object.metadata.name.includes(name);
```

Comment:
please also add condition for ready state:
```golang
        watchObj.object.status.conditions.some(
          (c) => c.type == "Ready" && c.status == "True"
        )
        ```

Reply:
added